apiversions.go

```diff
@@ -70,3 +71,26 @@ func (c *Client) ApiVersions(
 
 	return resp, err
 }
+
+// IsApiKeySupported checks if the API key is supported by the broker
+func (c *Client) IsApiKeySupported(
```

Comment:
I think we usually leave these checks to the users of the library

Reply:
I needed this to  stop the test from running on brokers that don't support this. Will make it a part of the tests. Let me know if you have a prettier approach for the same.

Reply:
If you know the exact versions of kafka you're targeting in the test you can use https://github.com/segmentio/kafka-go/blob/882ccd8dc16155638a653defe226d6492b0a9da8/testing/version.go#L39 and `t.Skip` when kafka version doesn't support it

Reply:
This is neat! Will use it. 👍